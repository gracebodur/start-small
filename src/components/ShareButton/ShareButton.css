.Facebook__share {
    border: none;
    padding: 0;
    background: none;
}

@media only screen 
and (min-device-width: 375px) 
and (max-device-width: 812px) 
and (-webkit-min-device-pixel-ratio: 3) { 
  .Facebook__share {
   width: 20%;
  }
}