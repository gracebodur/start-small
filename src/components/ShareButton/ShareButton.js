import React from 'react'

function ShareButton() {
    return(
        <section>
            <iframe 
                title='facebook'
                src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&layout=button&size=large&width=77&height=28&appId" 
                width="77" 
                height="28" 
                style={{border:'none'}} 
                scrolling="no" 
                frameBorder="0" 
                allowtransparency="true" 
                allow="encrypted-media"
                data-href="https://startsmall.now.sh/" >
            </iframe>
        </section>
    )
}

export default ShareButton