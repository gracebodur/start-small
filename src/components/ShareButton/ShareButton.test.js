import React from "react";
import toJson from "enzyme-to-json";
import ShareButton from "./ShareButton";

describe(`ShareButton component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a ShareButton by default", () => {
    const wrapper = <ShareButton />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the ShareButton given props", () => {
    const wrapper = <ShareButton {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
