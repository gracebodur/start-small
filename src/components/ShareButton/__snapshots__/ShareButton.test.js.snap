// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShareButton component renders a ShareButton by default 1`] = `undefined`;

exports[`ShareButton component renders the ShareButton given props 1`] = `undefined`;
